recipe/meta.yaml

```
@@ -9,25 +8,31 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 0c617dec19ed4f1ff6b247cd73855e949d87052d71900938b71f0cafd92f191b
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
+  skip: true  # [py<39]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - flit-core <4
   run:
-    - python >=3.7
-    - marshmallow >=3.0.0
-    - packaging
+    - python
+    - marshmallow >=3.13.0,<5.0.0
+    - packaging >=17.0
+    - typing_extensions >=4.0 # [py<310]
```

```suggestion
  run_constrained:
    - flask >=0.12.5
    - django >=2.2.0
    - bottle >=0.12.13
    - tornado >=4.5.2
    - pyramid >=1.9.1
    - falcon >=2.0.0
    - aiohttp >=3.0.8
```

I don't see typing_extentions upstream. We could add run_constrained